corehq/apps/export/static/export/js/utils.js

```diff
@@ -1,4 +1,16 @@
-hqDefine('export/js/utils.js', function () {
+if (!String.prototype.endsWith) {
```

Comment:
it's not supported at all in IE 😕 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#Browser_compatibility